UserGuide.md

```diff
@@ -110,12 +110,12 @@ int[] values = { 1 };
 gson.toJson(values);       // ==> [1]
 
 // Deserialization
-int one = gson.fromJson("1", int.class);
-Integer one = gson.fromJson("1", Integer.class);
-Long one = gson.fromJson("1", Long.class);
-Boolean false = gson.fromJson("false", Boolean.class);
```

Comment:
Thanks, yes, @lengyijun had the right idea here. Imagine calling a variable `false`!